doc/tutorials/using_celadon_as_uos.rst

```diff
@@ -0,0 +1,113 @@
+.. _using_celadon_as_uos:
+
+Using Celadon as User OS
+########################
+
+`Celadon <https://01.org/projectceladon/>`_ is an open source Android* software reference stack
+for Intel architecture. It builds upon a vanilla Android stack and incorporates open sourced components
+that are optimized for the hardware. This tutorial describes how to run Celadon as the User OS
+on the ACRN hypervisor. We are using Kaby Lake-based NUC (model NUC7i7DNHE) in this tutorial.
+Other :ref:`ACRN supported platforms <hardware>` should work as well.
+
+Introduction to OVMF
```

Comment:
I don't think it makes much sense to introduce OVMF here since we're only going to use a pre-compiled binary. I would get rid of this paragraph altogether.

Reply:
Sure, I will remove this part since the built-in OVMF is also work fine.